styles.css

```diff
@@ -0,0 +1,131 @@
+header {
+    height: 60px;
+    background-image: url(./images/headerphoto.jpg);
+}
+
+section {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    padding: 75px 175px;
+}
+
+section:nth-of-type(1), section:nth-of-type(3), section:nth-of-type(5) {
+    flex-direction: column;
+    background-size: cover;
+    background-attachment: fixed;
+    background-position: center;
+    height: 70vh;
+    align-items: flex-start;
+}
+
+section:nth-of-type(2), section:nth-of-type(4) {
+    background-color:white;
+    font-weight: 300;
+    font-style: normal;
+    font-size: 14px;
+    letter-spacing: .03em;
+    line-height: 1.8em;
+    text-transform: none;
+    color: rgba(0,0,0,.7);
+}
+
+section:nth-of-type(1) {
+    background-image: url(./images/headerphoto.jpg);
+}
+
+section:nth-of-type(3) {
+    background-image: url(./images/gallery-pexels-ali-pazani-2787341.jpg);
+    background-position: top;
+}
+
+section:nth-of-type(5) {
+    flex-direction: column;
+    background-image: url(./images/adam-winger-WXmHwPcFamo-unsplash.jpg);
+    align-items: center;
+}
+
+section:nth-of-type(6) {
+    flex-direction: column;
+    height: fit-content;
+    background-color:rgb(216,192,208);
+    color: white;
+    justify-content: center;
+    
+}
+
+section > div {
+    margin: 0 20px;
+    min-height: inherit;
+    min-width: 30vw;
+    max-width: 100vw;
+}
+
+.centered {
+    line-height: 2em;
+    text-align: center;
+}
+
+ul img {
+    width: 1em;
+    height: auto;
+    padding: 5px;
+}
+
+button {
+    border: 2px solid rgb(232,156,44);
+    color: rgb(232,156,44);
+    letter-spacing: .2em;
+    background-color: transparent;
+    padding: 10px 30px;
+}
+
+button:hover {
+    background-color: rgb(232,156,44);
+    color: white;
```

Comment:
add a cursor when hovering buttons so the user knows it's clickable. `cursor: pointer;`